chart/epinio/values.yaml

```
@@ -115,10 +115,10 @@ global:
   # The domain that will be used to access the epinio API server and the registry
   domain: ""
   # Connection details for the container registry.
-  registryURL: ""  # Skip if containerregistry.enabled is true
+  registryURL: "" # Skip if containerregistry.enabled is true
```

```suggestion
  registryURL: ""  # Skip if containerregistry.enabled is true
```
keep the space for the linter